scripts/linters/css_linter.py

```diff
@@ -108,7 +137,8 @@ def _lint_css_files(self):
         if num_files_with_errors:
             for error in result_list:
                 python_utils.PRINT(error)
-                summary_messages.append(error)
+            summary_messages.append(
+                self._get_trimmed_error_messages(result_list[0]))
```

Comment:
Why not do: "for result in result_list: summary_messages.append(self._get_trimmed_error_messages(result))"? That keeps the code general.

It seems to me that what you are doing here is to only handle the first filepath, if I'm reading correctly (since one line is added to result_list for each filepath, above).

Reply:
Done.